src/MessagingBubble/MessagingBubble.less

```diff
@@ -22,6 +23,14 @@
   white-space: pre-wrap;
 }
 
+.MessagingBubble--Link,
+.MessagingBubble--Link:focus,
+.MessagingBubble--Link:hover {
```

Comment:
not sure if the `:focus` and `:hover` are necessary if the CSS is the same as for `.MessagingBubble--Link`? But maybe so

Reply:
Without these it will still show the default link text-decoration on hover/focus (light blue, no underline) - I read that if you were to specify the element as well like `.MessagingBubble--Link **a**` then I wouldn't need to include the `:focus` and `:hover`, but I thought this was a bit clearer vs. specifying the `a` tag.